src/toil/cwl/cwltoil.py

```diff
@@ -3381,11 +3388,13 @@ def main(args: Union[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
                 # were required.
                 rm_unprocessed_secondary_files(param_value)
 
+            logger.debug('tool ', tool)
             try:
                 wf1, _ = makeJob(
```

Comment:
I'm not entirely sure what this code does.  It looks like it starts the workflow search so I'm not sure if cwl parsed the tool at this point.  I wasn't able to use the id key to prime this call with the correct workflow name.  So I just added workflow as a dummy variable.